clients/include/testing_gemm_batched_ex.hpp

```diff
@@ -15,7 +15,7 @@
 #include "unit.hpp"
 #include "utility.hpp"
 
-#define DEBUG_PRINT false
+#define DEBUG_PRINT 0
```

Comment:
If `DEBUG_PRINT` is used in statements like `#if DEBUG_PRINT`, then it needs to be a numerical value rather than `true` or `false` .